libs/services/designer-client-services/src/lib/consumption/manifests/swaggerFunctions.ts

```diff
@@ -0,0 +1,157 @@
+          'x-ms-deserialization': {
+            type: 'swaggeroperationid',
+            parameterReference: 'operationId',
+            options: {
+              swaggerOperation: {
+                methodPath: ['operationDetails', 'method'],
+                uriPath: ['operationDetails', 'uri'],
+              },
+            },
+          },
+          'x-ms-dynamic-list': {
+            dynamicState: {
+              operationId: 'getSwaggerFunctionOperations',
+            },
+            parameters: {
+              type: 'object',
+              functionAppId: {
+                parameterReference: 'functionApp.id',
+                required: true,
+              },
+            },
+          },
+        },
+        operationDetails: {
+          title: 'Operation Parameters',
+          description: 'Operation parameters for the above operation',
+          'x-ms-dynamic-properties': {
+            dynamicState: {
+              extension: {
+                operationId: 'getSwaggerFunctionOperationSchema',
+              },
+              isInput: true,
+            },
+            parameters: {
+              type: 'object',
+              operationId: {
+                parameterReference: 'operationId',
+                required: true,
+              },
+              functionAppId: {
+                parameterReference: 'functionApp.id',
+                required: true,
+              },
+            },
+          },
+        },
+      },
+      required: ['operationId'],
+    },
+    inputsLocationSwapMap: [{ source: ['operationDetails'], target: [] }],
+    inputsLocation: ['inputs'],
+    isInputsOptional: false,
+
+    outputs: {
+      type: 'object',
+      required: [],
+      properties: {
+        body: {
+          title: 'Body',
+          'x-ms-dynamic-properties': {
+            dynamicState: {
+              extension: {
+                operationId: 'getSwaggerFunctionOperationSchema',
+              },
+            },
+            parameters: {
+              type: 'object',
+              operationId: {
+                parameterReference: 'operationId',
+                required: true,
+              },
+              functionAppId: {
+                parameterReference: 'functionApp.id',
+                required: true,
+              },
+            },
+          },
+        },
+      },
+    },
+    isOutputsOptional: false,
+    includeRootOutputs: true,
+
+    customSwagger: {
+      // location: ['metadata', 'apiDefinitionUrl'],
+      operationId: 'getSwaggerFunctionUrl',
+      // 'x-ms-dynamic-properties': {
+      //   dynamicState: {
+      //     extension: {
+      //       operationId: 'getSwaggerFunctionUrl',
+      //     },
+      //   },
+      //   parameters: {
+      //     type: 'object',
+      //     functionAppId: {
+      //       parameterReference: 'functionApp.id',
+      //       required: true,
+      //     },
+      //   },
+      // },
```

Comment:
This is the format we use for the other schema/values clients, I initially tried to match them 1:1 but we use this data pre-parsing the parameters, so the info we normally have for the other clients we don't have yet (in getCustomSwaggerIfNeeded), wasn't really sure how to proceed so I'm just passing the whole definition instead of specific parameters